.editorconfig

```diff
@@ -103,6 +103,9 @@ csharp_prefer_braces = true:error
 #do not suggest readonly fields
 dotnet_style_readonly_field = false:error
 
+#prefer conditional expression over assignement
+dotnet_style_prefer_conditional_expression_over_assignment = true:error
```

Comment:
I think we should not enforce this. Can result in a very complex and unreadable expression in many cases. IMO it should be a suggestion.

Reply:
I removed the rule because if we add it even as a suggestion and we do not apply it everywhere we will get many suggestions and then we need to use `#pragma warning disable IDE0045 // Convert to conditional expression` to get rid of them, but that will be a lot.